src/base/types.hh

```diff
@@ -138,6 +138,66 @@ class Cycles
     friend std::ostream& operator<<(std::ostream &out, const Cycles & cycles);
 };
 
+class PcCountPair
```

Comment:
I think this should be in its own file. I would make a new file `src/cpu/probes/pc_count_pair.hh`